eq-author-api/schema/typeDefs.js

```diff
@@ -1029,4 +1029,10 @@ input DeleteCollapsibleInput {
 type Subscription {
   validationUpdated(id: ID!): Questionnaire!
 }
+
+input PublishQuestionnaireInput {
+  questionnaireId: ID
+  surveyId: String
+  formType: String
```

Comment:
Can we make all three of these inputs not nullable as we don't want the migration to work without all three.